main/src/main/scala/sbt/Keys.scala

```diff
@@ -486,7 +486,6 @@ object Keys {
   @cacheLevel(include = Array.empty)
   val bspReporter = taskKey[BuildServerReporter]("").withRank(DTask)
 
-  val useCoursier = settingKey[Boolean]("Use Coursier for dependency resolution.").withRank(BSetting)
```

Comment:
Is there any other key that was Ivy specific?

Reply:
It seems like there are some internal Ivy usage, but I could probably drop the external ivy.xml file support - 968bd9c7a2678aa6dcd16daa1fba3e9c2bc9f175